serverless-cljs-plugin/serverless_lumo/build.cljs

```diff
@@ -134,6 +140,8 @@
 
 (defn ^:export -main [& args]
   (let [opts (cli-options (cmd-line-args))]
-    (build! opts (merge-maps default-config (read-conf!)))))
+    (.then (build! opts (merge-maps default-config (read-conf!)))
+           lumo/exit
+           #(lumo/exit 2))))
```

Comment:
I put `2` in order to differentiate it from the normal exit code for throws, which is `1`.